cbor/src/main/java/com/fasterxml/jackson/dataformat/cbor/CBORGenerator.java

```diff
@@ -402,9 +417,20 @@ public final void writeStringField(String fieldName, String value)
             return;
         }
         _verifyValueWrite("write String value");
+        _decrementList();
         _writeString(value);
     }
 
+	
+    public final void writeFieldLong(long size) throws IOException {
```

Comment:
As I said, from my perspective only, this is not something that is required (as I use the CBORGenerator directly).

I only need to be able to create map and be able to use number as key.